yb-voyager/src/query/queryparser/ddl_processor.go

```diff
@@ -354,6 +375,9 @@ func (t *Table) addConstraint(conType pg_query.ConstrType, columns []string, spe
 	generatedConName := tc.generateConstraintName(t.TableName)
 	conName := lo.Ternary(specifiedConName == "", generatedConName, specifiedConName)
 	tc.ConstraintName = conName
+	if conType == FOREIGN_CONSTR_TYPE {
+		tc.ReferencedTable = lo.Ternary(referencedTable.Schemaname != "", referencedTable.Schemaname+"."+referencedTable.Relname, referencedTable.Relname)
```

Comment:
nit: i think this operation of fetching qualified name is used at multiple places.
Consider writing a helper/util function for this.